src/software/simulated_tests/tbots_test_runner.py

```diff
@@ -140,6 +140,15 @@ def set_play(self, play: Play, is_friendly: bool):
 
         fs_proto_unix_io.send_proto(Play, play)
 
+    def clear_world_buffer(self):
+        """
+        Clears the world buffer
+        """
+        world = self.world_buffer.get(return_cached=False)
+
+        while world is not None:
+            world = self.world_buffer.get(return_cached=False)
```

Comment:
Why is this needed? I noticed it being used in ball placement play test, though that test should have been fixed after your previous PR.